### ISSUE TITLE
Add information on no longer produced but still good flashcarts

### ISSUE THREAD

**Issue by mariomadproductions** [flashcarts/wiki#14]
E.g. the Acekard 2i.

**Comment by lifehackerhansol** [comment]
I feel like this is better for https://github.com/flashcarts/nds-index and not here.

We do need to do a better advertising for that second site, but as-is it's kind of janky so dunno.

**Comment by mariomadproductions** [comment]
I guess so, if that gets a page that lists the best flashcarts/gets a comparison page.

**Comment by lifehackerhansol** [comment]
I don't think a comparison is worth adding. It's meaningless when you can't get the big ones without selling an arm and a leg.

The goal of the quick start guide is to get users a recommendation on carts that are *currently sold new*. In general, if you already have a cart, and it isn't the original R4SDHC or any other broken variation of it (though we're kind of working on those), you don't need another flashcart, and I certainly don't want to encourage any more cash burnt on them either.

@Sanrax @Deletecat dunno what you think of it.

**Comment by Sanrax** [comment]
I think this is a good idea personally. Carts like AK2i and DSOne SDHC can still be obtained on the used market, and are better choices over currently recommended carts, if one is willing to spend the money needed.

**Comment by Deletecat** [comment]
I agree with lifehackerhansol to be honest. I would prefer if the quick start guide focused solely on flashcarts currently being sold new. They are often sold at a low price\*\* and are available from various different marketplaces.

With older flashcarts, it becomes a bit of a gamble. You can get good deals on some of them - for instance, I recently picked up a DSOne SDHC for £6 - but oftentimes people sell them for far more than they are actually worth due to their "rarity". 
With the DSOne mentioned before, I often see eBay listings for it hovering around £50 (⚠️). That is not something I would feel comfortable telling people to buy. 
While we could list *acceptable* price ranges for these old flashcarts, chances are you aren't going to find them within said range.

For the vast majority of people, the flashcarts currently listed on the quick start guide do the job just fine. They aren't perfect by any means, but they aren't so bad that I would consider telling people to go find a used cart.

---

\*\*The exceptions to this are the EX4DS and EZ Flash Parallel - but these aren't really flashcarts that we recommend people go out and buy in the first place.

**Comment by mariomadproductions** [comment]
They don't necessarily have to be listed as totally equivalent to the ones you can buy new, but I thinks its still a good idea to list them, as the fact that these cards are the best option in terms of compatibility seems to be missing from a lot of sites. And I think compatibility info is pretty important.

**Comment by Deletecat** [comment]
I've changed my mind about this in recent times. After the couple incidents of China-US shipping being halted and tariffs having doubled the price of these carts temporarily, hosting information about second-hand carts is probably a good idea. They won't always be readily available (or have an [acceptable price](https://www.ebay.co.uk/itm/267075407707)...) but people can make their own decisions based on the info provided.

Coming back to it today where just about all of our recommended carts have just disappeared from AliExpress. There are still tons of listings of Ace3DS+ clones, and a couple Demon carts kicking about, but whether more listings will disappear... I'm not sure. Being able to recommend decent second hand carts though would help the situation slightly.
